### DISCUSSION TITLE
Using pastas to simulate piezometers response to pumping in a mine site.

### DISCUSSION THREAD

**Discussion by BenUq** [pastas/pastas#653] [Q&A]
I am reaching out to seek your valuable insights on a few matters regarding using pastas to simulate piezometer response to pumping, snow melt, and atmospheric pressure change in a mine site: 

1. I have piezometer observation time series in ft and pumping rates in gallons per minute, do I have to converts the units of measurement to millimeters and mm3/day? or I can use feet and GPM directly?

2. In the context of changes in atmospheric pressure, I am interested in knowing which response function you would recommend or advise for our analysis.
 
3.	My current dataset comprises data from more than 10 pumping wells. I am looking for guidance on how to accurately compute the individual impact of each pumping well on the piezometer responses.

**Comment by BenUq** [comment]
Any help ?????

**Comment by dbrakenhoff** [comment]
Hi @BenUq,

Sorry for the delay, it's been busy for the core developers with other work, so we've been unable to get to your question until now. Also note that our priorities go out to bugs and issues with the code, and user questions are something we answer in our "free" time, so sometimes will take a bit longer to get to. I hope you understand. 

1. The units generally do not matter. If your pumping units are GPM, then the gain of the response function for pumping will be in ft / GPM. Just be sure to interpret your results in the correct units (i.e. the ones defined in your input time series). One note, if you use a non-linear RechargeModel, the units must be mm/day. Otherwise you're free to choose whichever units are most convenient for you. 
2. For atmospheric pressure, I would guess the `Exponential` response would suffice. There is generally quick relationship between atmospheric pressure and measured heads? If you think there is lag between the two, consider using `Gamma`. 
3. When computing the individual impact of each well, adding 10 wells individually could lead to overfitting (the model will start using some of the well pumping rate time series to fit remaining trends in the observed heads). I recommend taking a look at the [notebook about adding multiple wells](https://pastas.readthedocs.io/en/master/examples/multiple_wells.html). But definitely try both approaches and compare the results. My guess is the `WellModel` will provide a more realistic estimate. But your results will really depend on the variability in each pumping time series, the length of your time series series, etc. It will take some experimentation and judgment on your part to determine whether your results are plausible. 

I also, recommend browsing through the [documentation](https://pastas.dev). There are a lot of examples there that could be useful. 

Cheers,

Davíd

**Reply by BenUq** [reply]
hi @dbrakenhoff, 
 thanks very much for your response.   I Will take a look to the suggested documentation, and I hope to share some of the results.

**Reply by BenUq** [reply]
@dbrakenhoff

**Comment by BenUq** [comment]
Could anyone help me to figure out what is going with my results. The attached is show the pasta overestimate the hydraulic head when I simulate additional one year from calibration period. My calibration period with tmin = 2020 and tmax =2022 gives R = 0.98 but when simulating to 2023 R = 0.55.
![PZ_simulate](https://github.com/pastas/pastas/assets/147563014/93f26e71-d086-46f7-9ff7-80d7bfdff94d)

**Reply by BenUq** [reply]
Here is the fit statistics. 
<img width="388" alt="image" src="https://github.com/pastas/pastas/assets/147563014/e93374b3-792c-4d46-9e7c-54c1ff4ad48a">

**Reply by dbrakenhoff** [reply]
Hi @BenUq,

It's difficult to say why you're getting that result. Is the pumping for your first well at the end of 2022 about the same or slightly higher than in early 2021? The calculated contribution suggests this is the case whereas your head seems to show a bigger drop in 2022. Maybe you're missing an influence that could have further lowered the head in this period? 

One thing to note is that fit report is generating the warning that the response time for the first well is longer than the calibration period, meaning the influence of pumping on day 1 in your time series still has an effect on the last day in your calibration series. This means there is significant uncertainty about the tail of your response, and you should be careful in interpreting the parameters of that response function. 

Anyway, hope this helps you think about the results a little... 

Kind regards, 

Davíd

**Reply by BenUq** [reply]
Hi @dbrakenhoff 
![image](https://github.com/pastas/pastas/assets/147563014/0e0d8f28-782c-4a1e-a6b4-48860bbacef8)

Thanks for this insightful reply, 
Yes, near to the end of 2022 the pump (PW-334) was down (0 GPM) that is why the observed water level start rising again at the end of 2022. Also, the pumping rate on average higher between October 2021 - October 2021 just before the PW-334 went down by 140 GPM. I do agree with you that I might be missing another influence.  

I did not figure how to address the warning generated from the fit report. Is a way of addressing the warning so that it does not affect the results?

Kind regards 
Ben
